renderdoc/driver/d3d12/d3d12_pixelhistory.cpp

```diff
@@ -573,25 +573,50 @@ struct D3D12PixelHistoryCallback : public D3D12ActionCallback
       baseSlice = 0;
     }
 
-    // For pipeline barriers.
```

Comment:
Code readability/consistency : I would be tempted to leave this here
and in the MSAA branch copy `barrier` into `barriers[0]`  and if any MSAA specific changes are needed do them in place in `barriers[0]`

Reply:
Will do.